client/command.go

```diff
@@ -4,7 +4,8 @@ import (
 	"fmt"
 	"io"
 	"os"
-	"os/exec"
+
+	exec "golang.org/x/sys/execabs"
```

Comment:
oh, nice; I missed that they included it there as well, I thought it was only in an `internal/` package (https://github.com/golang/go/commit/07e3195293ec510171d7d43ec8ac2bcb9cf00df#diff-25560022006dcc0a0bf07fe3291119c24ca8b12d0f29d3ad01f7c8a55265af58R34-R43)